OsmAnd-java/src/main/java/net/osmand/router/RoutingContext.java

```diff
@@ -676,6 +678,10 @@ public boolean checkIfMemoryLimitCritical(long memoryLimit) {
 	}
 	
 	public void unloadUnusedTiles(long memoryLimit) {
+		// TODO DELETE this check (test with & without it carefully !)
```

Comment:
// TODO

Reply:
all tests with & without (Java and C++) ok